.pre-commit-config.yaml

```diff
@@ -30,7 +30,7 @@ repos:
         language: system
         files: '\.(ncl|NCL)$'
   - repo: https://github.com/lorenzwalthert/precommit/  # Checks for R
-    rev: 'v0.3.2.9007'
+    rev: 'v0.1.3'
```

Comment:
Could you propose this change in a separate pull request, please? And describe clearly why it is needed? The most recent release appears to be 0.4.2: https://github.com/lorenzwalthert/precommit/releases

Reply:
> Could you propose this change in a separate pull request, please? And describe clearly why it is needed? The most recent release appears to be 0.4.2: https://github.com/lorenzwalthert/precommit/releases

I have reversed this change for this issue in commit https://github.com/ESMValGroup/ESMValTool/pull/3652/commits/05793d6082fc901e93dd33e1f8d42b33b40b6c16 and have made a new issue so i can work on it (and will make a new PR) as you have asked :slightly_smiling_face:  Thanks Bouwe

Reply:
Thanks!